pkg/workflows/sdk/runtime.go

```diff
@@ -30,11 +30,13 @@ type Runtime interface {
 }
 
 type FetchRequest struct {
-	URL       string            `json:"url"`                 // URL to query, only http and https protocols are supported.
-	Method    string            `json:"method,omitempty"`    // HTTP verb, defaults to GET.
-	Headers   map[string]string `json:"headers,omitempty"`   // HTTP headers, defaults to empty.
-	Body      []byte            `json:"body,omitempty"`      // HTTP request body
-	TimeoutMs uint32            `json:"timeoutMs,omitempty"` // Timeout in milliseconds
+	URL         string            `json:"url"`                   // URL to query, only http and https protocols are supported.
+	Method      string            `json:"method,omitempty"`      // HTTP verb, defaults to GET.
+	Headers     map[string]string `json:"headers,omitempty"`     // HTTP headers, defaults to empty.
+	Body        []byte            `json:"body,omitempty"`        // HTTP request body
+	TimeoutMs   uint32            `json:"timeoutMs,omitempty"`   // Timeout in milliseconds
```

Comment:
Things we need to think about with this change:
- Does TimeoutMs apply to all the requests including timeout?
- How do we know how long to backoff for (in the server side)?